### PR TITLE
add login url and token check

### PR DESCRIPTION
**Pull request by washort**

### CODE REVIEW

**Comment by openjck** [comment]
Looks like tests are failing because the _psycopg2_ package was commented.

File: distributionviewer/api/views.py [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 from rest_framework.response import Response
 from rest_framework.renderers import JSONRenderer
 
+from distributionviewer.authentication import commonplace_token
 from .models import CategoryCollection, LogCollection, Metric
 from .renderers import MetricsJSONRenderer
 from .serializers import (CategoryDistributionSerializer,
                           LogDistributionSerializer, MetricSerializer)
 
+from oauth2client import client, crypt
```
Comment (robhudson) [inline distributionviewer/api/views.py]: I would think this would go with the other 3rd party imports?

File: distributionviewer/urls.py [hunk 2]
```diff
@@ -2,13 +2,14 @@
...
 
-from .api.views import metric, metrics
+from .api.views import login, metric, metrics
 
 
 urlpatterns = [
     url(r'^metrics/$', metrics, name='metrics'),
     url(r'^metric/(?P<metric_id>\d+)/$', metric, name='metric'),
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^login/', login, name='login'),
```
Comment (robhudson) [inline distributionviewer/urls.py]: Probably want this to be `r'^login/$'` (with the ending `$`).

**Comment by robhudson** [comment]
r+wc

**Pull request merged**